packages/grid/_modules_/grid/hooks/features/column-reorder/useColumnReorder.tsx

```diff
@@ -1,24 +1,21 @@
 import * as React from 'react';
-import { ColDef } from '../../models/colDef';
-import { useLogger } from '../utils';
-import { ApiRef } from '../../models';
+import { ColDef } from '../../../models/colDef';
+import { useLogger } from '../../utils';
+import { ApiRef } from '../../../models';
```

Comment:
For the import at the hooks level, better to put the full path, this is to avoid cyclic dependencies